README.md

```diff
@@ -62,7 +71,7 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-Your answer...
+It allows us to export code from one file to another.   We use it to separate our js files by exposing their content as one global variable.
 ```
 
```

Comment:
It only exposes what you tell to it expose as an encapsulated reference and not a global variable. The whole pattern of modularization implemented in Node via `module.exports` and `require()` is there to not only help organize code, but to avoid globals.